### PR TITLE
Move test-only dependencies into devDependencies

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,40 +64,23 @@
     "hapi": "11.1.2",
     "hatchet": "0.3.2",
     "hoek": "3.0.4",
-    "imagemin-cli": "2.0.0",
     "imports-loader": "0.6.5",
     "inert": "3.2.0",
-    "inject-loader": "2.0.1",
```
Comment (jbuck) [inline package.json]: This was removed entirely because it's not being used

File: package.json [hunk 2]
```diff
@@ -67,40 +64,23 @@
...
-    "karma-coverage": "0.5.3",
-    "karma-coveralls": "1.1.2",
-    "karma-firefox-launcher": "0.1.7",
-    "karma-mocha": "0.2.1",
-    "karma-mocha-reporter": "1.1.3",
-    "karma-sourcemap-loader": "0.3.6",
-    "karma-webpack": "1.7.0",
     "less": "2.5.3",
     "less-loader": "2.2.2",
-    "mkdirp": "0.5.1",
```
Comment (jbuck) [inline package.json]: This was removed entirely because it's not being used

File: package.json [hunk 3]
```diff
@@ -67,40 +64,23 @@
...
-    "mocha": "2.3.4",
     "newrelic": "1.24.0",
     "npm-run-all": "1.4.0",
     "polyfill-service": "2.2.0",
-    "postcss-loader": "0.8.0",
```
Comment (jbuck) [inline package.json]: This was removed entirely because it's not being used

**Comment by jbuck** [comment]
@alicoding r?

**Comment by jbuck** [comment]
@ScottDowne r?

**Comment by alicoding** [comment]
r+ from me. I've tested it and works perfectly locally.

**Pull request merged**